src/devices/OneWire/OneWireBus.cs

```diff
@@ -62,8 +62,11 @@ public IEnumerable<OneWireDevice> EnumerateDevices(DeviceFamily family = DeviceF
         /// <param name="numDevices">Max number of devices to scan for before finishing.</param>
         /// <param name="numScans">Number of scans to do to find numDevices devices.</param>
         /// <returns>Task representing the async work.</returns>
-        public Task ScanForDevicesAsync(int numDevices = 5, int numScans = -1)
+        public Task ScanForDevicesAsync(int numDevices = 64, int numScans = -1)
         {
+            // Default 64 used to align with Linux driver
+            // https://github.com/torvalds/linux/blob/v5.3/drivers/w1/w1.c#L46
```

Comment:
with this link now we essentially must remove this reference unless you find public API mention about 64 👀 (sorry, didn't invent these rules and also not a fan of it)